docs/HowToUsePeripheral.md

```diff
@@ -423,18 +437,18 @@ Please refer to **[i2c.h](../os/include/tinyara/i2c.h)** to know more about the
 
 ## I2S
 
-I2S port for a board involves the following;
-1. I2S Device Structure and its Operations.
+I2S port for a board involves the following;  
```

Comment:
Convention for directing attention to a bullet list is colon (':'), as described [here](https://writingcenter.unc.edu/tips-and-tools/semi-colons-colons-and-dashes/). Also, please remove space at the end (also in point 1).

Reply:
Ok I will change it.
the spaces mean 'change lines', It is one of markdown syntax.